content/documentation/glossary/concepts/index.adoc

```diff
@@ -0,0 +1,105 @@
+---
+title: "Concepts"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Concepts Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a logical group of <<Workload>> defined by the <<Label App>>.
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<Label App>> on the <<Workload>>
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+
+
+== Istio Config
+
+It's a configuration object in an Istio environment
+
+
+=== Istio object/configuration Type
+
+It's the type of the <<Istio Config>>, could be <<Gateway>>, <<VirtualService>>, <<DestinationRule>>, <<ServiceEntry>>, <<Rule>>, <<QuotaSpec>> or <<QuotaSpecBinding>>.
+
+== Istio Sidecar
+
+Read the <<Istio Sidecar>> definition in link:https://istio.io/docs/reference/commands/sidecar-injector/[Istio Sidecar].
+
+== Quota
+
+A limited or fixed number or amount of resources.
+
+== QuotaSpec
+
+File that define a <<Quota>>
+
+== QuotaSpecBinding
+
+Quota spec binding is used to bind link:https://istio.io/help/faq/mixer/[mixer] quota resource name to a proxy.
+
+== Label
+
+It's a tag assigned by the client/user to identify a set of objects.
+
+An empty link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (that is, one with zero requirements) selects every object in the collection.
+
+A null link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (which is only possible for optional selector fields) selects no objects.
+
+For example, Istio uses the <<Label App>> & <<Label Version>> in a <<Workload>> to specify the version and the application.
+
+=== Label App
+
+This is the 'app' label on an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Label Requirements]
+
+=== Label Version
+
+This is the 'version' label on an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Label Requirements]
+
+
+== Namespace
+
+Namespaces are intended for use in environments with many users spread across multiple teams, or projects
+
+Namespaces are a way to divide cluster resources between multiple users
+
+== ReplicaSet
+
+Ensures that a specified number of pod replicas are running at any one time
+
+== Service
+
+Pods are dynamically created, they are mortal. A <<Service>> is an abstraction which defines a logical set of Pods and a policy by which to access them.
```

Comment:
I'm not sure that `mortal` is the best word here. Probably the whole  `they are mortal` can be eliminated.

Reply:
"they are ephemeral" is what I think you mean here (not mortal).

Reply:
I mean kubernetes definition , It's similar https://kubernetes.io/docs/concepts/services-networking/service/